tests/build-viewer-reports

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+# Set emacs mode to Python
+# -*- mode: Python;-*-
```

Comment:
Why is this necessary? Probably to make `emacs` users happy, which is fine 😄

Reply:
It is important to make emacs users happy...